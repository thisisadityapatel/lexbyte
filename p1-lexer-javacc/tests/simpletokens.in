<<=>>=!====in!in:=
+-*/&|!
()[],;#1#0..
