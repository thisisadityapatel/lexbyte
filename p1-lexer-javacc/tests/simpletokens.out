String scanned: [<]
	->Token kind: "<"
	->Token value: null
String scanned: [<=]
	->Token kind: "<="
	->Token value: null
String scanned: [>]
	->Token kind: ">"
	->Token value: null
String scanned: [>=]
	->Token kind: ">="
	->Token value: null
String scanned: [!=]
	->Token kind: "!="
	->Token value: null
String scanned: [==]
	->Token kind: "=="
	->Token value: null
String scanned: [=in]
	->Token kind: "=IN"
	->Token value: null
String scanned: [!in]
	->Token kind: "!IN"
	->Token value: null
String scanned: [:=]
	->Token kind: ":="
	->Token value: null
String scanned: [+]
	->Token kind: "+"
	->Token value: null
String scanned: [-]
	->Token kind: "-"
	->Token value: null
String scanned: [*]
	->Token kind: "*"
	->Token value: null
String scanned: [/]
	->Token kind: "/"
	->Token value: null
String scanned: [&]
	->Token kind: "&"
	->Token value: null
String scanned: [|]
	->Token kind: "|"
	->Token value: null
String scanned: [!]
	->Token kind: "!"
	->Token value: null
String scanned: [(]
	->Token kind: "("
	->Token value: null
String scanned: [)]
	->Token kind: ")"
	->Token value: null
String scanned: [[]
	->Token kind: "["
	->Token value: null
String scanned: []]
	->Token kind: "]"
	->Token value: null
String scanned: [,]
	->Token kind: ","
	->Token value: null
String scanned: [;]
	->Token kind: ";"
	->Token value: null
String scanned: [#1]
	->Token kind: "#1"
	->Token value: null
String scanned: [#0]
	->Token kind: "#0"
	->Token value: null
String scanned: [..]
	->Token kind: ".."
	->Token value: null
